### PR TITLE
[DNM] Upgrades webhook info

### PR DESCRIPTION
**Pull request by quardbreak**
* Adds new webhook - a round preparation. Calls when Master fully initialized, so players can know when they're can join before round start. 
  * Now can click on the server link to join!
* Upgrades round end webhook. Now it show some information like players amount and round duration, plus, little format design.

Require some help to test this. I dunno how to test it in local environment.

### CODE REVIEW

File: code/modules/webhooks/webhook_roundstart.dm [hunk 1]
```diff
@@ -4,13 +4,11 @@
 // Data expects a "url" field pointing to the current hosted server and port to connect on.
 /decl/webhook/roundstart/get_message(var/list/data)
 	. = ..()
-	var/desc = "A new round is starting"
-	if(data && data["url"])
-		desc += " on [data["url"]]"
-	desc += "."
+	var/desc = "Gamemode: [SSticker.master_mode]\n"
+	desc += "Players: [GLOB.player_list.len]"
```
Comment (MistakeNot4892) [inline code/modules/webhooks/webhook_roundstart.dm:8]: No URL?

File: code/modules/webhooks/webhook_submap.dm [hunk 2]
```diff
@@ -7,10 +7,10 @@
 	var/submap_name = data && data["name"]
 	if(submap_name)
 		submap_name = ", [submap_name],"
-	var/desc = "A submap[submap_name] is currently available."
+	var/desc = "A submap[submap_name] is currently available"
```
Comment (MistakeNot4892) [inline code/modules/webhooks/webhook_submap.dm]: I can more or less agree with removing the periods from titles, but I really don't think removing them from the contents is good. It's a full sentence, it should be grammatically correct.

File: code/modules/webhooks/webhook_roundprep.dm [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/decl/webhook/roundprep
+	id = WEBHOOK_ROUNDPREP
+
+// Data expects a "url" field pointing to the current hosted server and port to connect on.
+/decl/webhook/roundprep/get_message(var/list/data)
+	. = ..()
+	var/desc = "The server has been started!\n"
+	desc += "Map: **[station_name()]**\n"
+	desc += "Adress: <[get_world_url()]>"
```
Comment (MistakeNot4892) [inline code/modules/webhooks/webhook_roundprep.dm]: Address.

File: code/modules/webhooks/webhook_roundstart.dm (hunk 1 above)
Comment (quardbreak) [reply code/modules/webhooks/webhook_roundstart.dm:8]: ~~Not sure what people want to join in game from roundend webhook. I made special hook where you can just click and join. Can add if you want.~~ Sorry, didn't see the context. 
Btw, think this PR need some discussion how those webhooks must look like.

**Comment by MistakeNot4892** [comment]
Feel free to reopen when ready for merge.

**Pull request closed without merging**